src/app/selectors/characters.js

```diff
@@ -96,3 +100,13 @@ export const charactersSortedAtoZSelector = createSelector(
   allCharactersSelector,
   (characters) => sortBy(characters, 'name')
 )
+
+export const charactersSortedInBookSelector = createSelector(
```

Comment:
good idea. I wouldn't have thought of this. We just can't forget to mirror this to the pltr repo